src/main/java/mate/jdbc/service/CarService.java

```diff
@@ -0,0 +1,23 @@
+package mate.jdbc.service;
+
+import java.util.List;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+
+public interface CarService {
+    Car create(Car car);
+
+    Car get(Long id);
+
+    List<Car> getAll();
+
+    Car update(Car car);
+
+    boolean delete(Long id);
+
+    void addDriverToCar(Driver driver, Car car);
+
+    void removeDriverFromCar(Driver driver, Car car);
```

Comment:
Consider returning the updated Car object after removing the driver from the car. Similar to the previous suggestion, this can help in chaining actions and provides better feedback to the caller.